IdentityCore/src/pop_manager/MSIDDevicePopManager.m

```diff
@@ -69,13 +69,49 @@ - (MSIDAssymetricKeyPair *)keyPair
     return _keyPair;
 }
 
+- (NSDictionary *)buildPayloadDict:(NSString *)accessToken
+                              host:(NSString *)host
+                        httpMethod:(NSString *)httpMethod
+                             nonce:(NSString *)nonce
+                              path:(NSString *)path
+                     publicKeyDict:(NSDictionary *)publicKeyDict
+{
+    NSMutableDictionary *payload = [NSMutableDictionary new];
+    
+    [payload setObject:accessToken forKey:@"at"];
+    NSDictionary *cnf = @{@"cnf": @{
+                                  @"jwk":publicKeyDict
```

Comment:
This will crash if publicKeyDict is nil. Can you ensure it's not-nil?

Reply:
Yes it's guaranteed not nil by the calling method 
```
- (NSString *)createSignedAccessToken:(NSString *)accessToken
                           httpMethod:(NSString *)httpMethod
                           requestUrl:(NSString *)requestUrl
                                nonce:(NSString *)nonce
                                error:(NSError *__autoreleasing * _Nullable)error
```